release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1095,6 +1095,38 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, a pod selected by an `EgressIp` object that was hosted on a secondary network interface controller (NIC) caused connections to node IP addresses to timeout. With this release, the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-26979[*OCPBUGS-26979*])
+
+* Previously, a leap file package that the {product-title} Precision Time Protocol (PTP) Operator installed could not be used by the `ts2phc` process because the package expired. With this release, the leap file package is updated to read leap events from Global Positioning System (GPS) signals and update the offset dynamically so that the expired package situation no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-25939[*OCPBUGS-25939*])
+
+* Previously, pods assigned an IP from the pool created by the Whereabouts CNI plugin were getting stuck in the `ContainerCreating` state after a node forced a reboot. With this release, the Whereabouts CNI plugin issue associated with the IP allocation after a node force reboot is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-24608[OCPBUGS-24608])
+
+* Previously, there was a conflict between two scripts on {product-title} in IPv6, including single and dual-stack, deployments. One script set the hostname to a fully qualified domain name (FQDN) but the other script might set it to a short name too early. This conflict happened because the event that triggered setting the hostname to FQDN might run after the script that set it to a short name. This occurred due to asynchronous network events. With this release, new code has been added to ensure that the FQDN is set properly. This new code ensures that there is a wait for a specific network event before allowing the hostname to be set. (link:https://issues.redhat.com/browse/OCPBUGS-22324[*OCPBUGS-22324*])
+
+* Previously, if a pod selected by an `EgressIP` through a secondary interface had its label removed, another pod in the same namespace would also lose its `EgressIP` assignment, breaking its connection to the external host. With this release, the issue is fixed, so that when a pod label is removed and it stops using the `EgressIP`, other pods with the matching label continue to use the `EgressIP` without interruption. (link:https://issues.redhat.com/browse/OCPBUGS-20220[*OCPBUGS-20220*])
+
+* Previously, `EgressIP` pods hosted by a secondary interface would not failover because of a race condition. Users would receive an error message indicating that the `EgressIP` pod could not be assigned because it conflicted with an existing IP address. With this update, the `EgressIP` pod moves to an egress node. (https://issues.redhat.com/browse/OCPBUGS-20209[(*OCPBUGS=20209*])
```

Comment:
```suggestion
* Previously, `EgressIP` pods hosted by a secondary interface would not failover because of a race condition. Users would receive an error message indicating that the `EgressIP` pod could not be assigned because it conflicted with an existing IP address. With this update, the `EgressIP` pod moves to an egress node. (https://issues.redhat.com/browse/OCPBUGS-20209[*OCPBUGS-20209*])
```

Reply:
:heavy_check_mark: